src/main/resources/theme/play/ranked_1v1.fxml

```diff
@@ -99,7 +99,6 @@
                       <Label styleClass="h2" text="%ranked1v1.rating.title" />
                       <Label fx:id="ratingLabel" prefHeight="80.0" styleClass="h1" text="0000" GridPane.halignment="CENTER" GridPane.rowIndex="2" GridPane.valignment="CENTER" />
                       <ProgressIndicator fx:id="ratingProgressIndicator" prefWidth="244.0" progress="0.66" GridPane.rowIndex="2" />
-                      <Label fx:id="ratingHintLabel" text="&lt;rating hint&gt;" wrapText="true" GridPane.rowIndex="1" />
```

Comment:
The controller needs to be cleaned up as well
https://github.com/FAForever/downlords-faf-client/blob/db7fa456e48c795c7cae45f9c87e47e08cf44d60/src/main/java/com/faforever/client/rankedmatch/Ladder1v1Controller.java#L259